Makefile

```diff
@@ -15,8 +15,6 @@
 # Various versions - Minishift, default OpenShift, default B2D ISO
 MINISHIFT_VERSION = 1.3.1
 OPENSHIFT_VERSION = v1.5.1
-B2D_ISO_VERSION = v1.0.2
-CENTOS_ISO_VERSION = v1.1.0
```

Comment:
So we would need something like

    B2D_ISO_BASE_VERSION = ~1		
    CENTOS_ISO_BASE_VERSION = ~1

This would instruct Miinishift to automatically download ISO >= 1, < 2